x/ccv/provider/keeper/throttle.go

```diff
@@ -29,9 +29,25 @@ func (k Keeper) HandlePendingSlashPackets(ctx sdktypes.Context) {
 	// Iterate through ordered (by received time) slash packet entries from any consumer chain
 	k.IteratePendingSlashPacketEntries(ctx, func(entry providertypes.SlashPacketEntry) (stop bool) {
 
+		// Obtain validator from consensus address.
+		// The slash packet validator address may be known only on the consumer chain
+		// in this case, it must be mapped back to the consensus address on the provider chain
+		consumerConsAddr := sdktypes.ConsAddress(entry.ValAddr)
```

Comment:
`entry.ValAddr` could be set to the validator operator address in `OnRecvSlashPacket`. That's where I'd also remap consumer addresses to provider addresses before queueing `PendingSlashPacketData`.

Reply:
SGTM, changes done in https://github.com/cosmos/interchain-security/pull/565/commits/f2a5522a4dfcd7025129ab6b264af7adb6f4fcb1. Note this makes the PR include refactors as well. So naming is changed accordingly